test/integration/integration_admin_test.cc

```diff
@@ -415,7 +417,7 @@ class IntegrationAdminIpv4Ipv6Test : public HttpIntegrationTest, public testing:
 public:
   IntegrationAdminIpv4Ipv6Test()
       : HttpIntegrationTest(Http::CodecClient::Type::HTTP1, Network::Address::IpVersion::v4,
-                            realTime()) {}
+                            simTime()) {}
```

Comment:
Did you mean to change this or should it be StatsMatcherIntegrationTest?

Reply:
Good catch, I mistakenly changed this on IntegrationAdminIpv4Ipv6Test when I meant to change it on StatsMatcherIntegrationTest. Will fix in 752b236.